src/frontend/sider/Fagsak/Personlinje/Behandlingsmeny/AInntekt/AInntekt.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { Alert, Dropdown, Loader } from '@navikt/ds-react';
+
+import { useHentAInntektUrl } from '../../../../../hooks/useHentAInntektUrl';
+import type { IMinimalFagsak } from '../../../../../typer/fagsak';
+
+interface IProps {
+    minimalFagsak: IMinimalFagsak;
+}
+
+export const AInntekt: React.FC<IProps> = ({ minimalFagsak }) => {
```

Comment:
`React.FC<IProps>` er ikke lenger nødvendig så vidt jeg vet, er vel blitt mer normalt å bare definere props slik som her:

```suggestion
export const AInntekt = ({ minimalFagsak }: Props) => {
```

Se f.eks: https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components/

> With latest React types and TypeScript 5.1. it's mostly a stylistic choice, otherwise discouraged.

Reply:
Jeg har blitt veldig vandt til å gjøre det på den gamle måten fra koden i søknad og tenkte ikke over at dette var en mulighet, men det ser langt bedre ut å gjøre det på den måten ja, så jeg legger det til 👍